openglyph/include/openglyph/steam/steam_paths.hpp

```diff
@@ -0,0 +1,32 @@
+#include <cstdint>
+#include <filesystem>
+#include <vector>
+namespace openglyph::steam {
+/**
+ * @brief this class groups alls steam path functions
+ */
+class SteamPaths
+{
+private:
+    /// @brief Returns the usual Steam root installation path for the current platform.
```

Comment:
"usual", or "actual"?
"C:\Program Files\Steam" might be the _usual_ path, but it may not be the actual path on this machine.
If you meant the latter, "for the current platform" is unnecessary.